client/src/components/App.jsx

```diff
@@ -40,11 +40,14 @@ const App = props => {
 
   useEffect(() => {
     document.body.style.zoom = '100%';
-    if (authenticated) {
-      fetchObservations(userSession);
-      fetchDemographicsComorbidities(userSession);
-    }
-  }, [fetchObservations, fetchDemographicsComorbidities, authenticated, userSession]);
+    const fetchData = async function () {
+      if (authenticated) {
+        fetchObservations(userSession);
+        await fetchDemographicsComorbidities(userSession);
+      }
+    };
+    fetchData();
+  }, [fetchObservations, fetchDemographicsComorbidities, authenticated, userSession, showOnboard]);
```

Comment:
```suggestion
  }, [fetchObservations, fetchDemographicsComorbidities, authenticated, userSession]);
```

This `useEffect` isn't dependent on `showOnboard`, so `showOnboard` shouldn't be in the dependencies list